### PR TITLE
Few Typo fixes

### PR DESCRIPTION
**Pull request by vasanth-selvaraj**

### CODE REVIEW

**Review by vasanth-selvaraj — COMMENTED** [review]
Fixed typos and spelling for the below files. Typos only fixed for the comments on the files

simclusters-ann/README.md
src/java/com/twitter/search/README.md
src/thrift/com/twitter/search/common/ranking/ranking.thrift
src/thrift/com/twitter/search/earlybird/thrift/earlybird.thrift
src/thrift/com/twitter/simclusters_v2/abuse.thrift
src/thrift/com/twitter/simclusters_v2/embedding.thrift
src/thrift/com/twitter/simclusters_v2/evaluation.thrift
src/thrift/com/twitter/simclusters_v2/identifier.thrift

File: src/java/com/twitter/search/README.md
Comment (aMiss-aWry) [inline src/java/com/twitter/search/README.md:1]: This is an incorrect fix. "sharded" is the correct term.

Comment (vasanth-selvaraj) [reply src/java/com/twitter/search/README.md:1]: Corrected

**Review by vasanth-selvaraj — COMMENTED** [review]
Reverted the incorrect fix

**Comment by Imfaqngroovn** [comment]
You had me at the "sharded" typo.

**Review by aMiss-aWry — COMMENTED** [review]
LGTM

**Review by Rahul-Prasad-07 — COMMENTED** [review]
Looks good

**Pull request closed without merging**